app/javascript/src/LoginPage/loginForms/Login3scaleForm.jsx

```diff
@@ -17,11 +17,14 @@ import {
 } from '@patternfly/react-core'
 
 type Props = {
-  providerSessionsPath: string
+  providerSessionsPath: string,
+  session: {
+    username: ?string
+  }
 }
 
 type State = {
-  username: string,
+  username: ?string,
```

Comment:
I think this is not necessary. Its value is not going to be `undefined`.

Reply:
True, good catch